nodl_python/nodl/_parsing/_parsing.py

```diff
@@ -34,16 +34,30 @@ def _parse_interface(interface: etree._Element) -> List[Node]:
 
 
 def _parse_element_tree(element_tree: etree._ElementTree) -> List[Node]:
-    """Extract an interface element from an ElementTree if present."""
+    """Extract an interface element from an ElementTree if present.
+
+    :param element_tree: parsed xml tree to operate on
+    :type element_tree: etree._ElementTree
+    :raises errors.InvalidNoDLDocumentError: raised if tree does not adhere to schema
+    :return: List of NoDL nodes present in the xml tree.
+    :rtype: List[Node]
+    """
     try:
         interface_schema().assertValid(element_tree)
     except etree.DocumentInvalid as e:
         raise errors.InvalidNoDLDocumentError(e)
     return _parse_interface(element_tree.getroot())
 
 
-def parse(path: Union[str, Path, IO]) -> List[Node]:
-    """Parse the nodes out of a given NoDL file."""
+def parse(*, path: Union[str, Path, IO]) -> List[Node]:
```

Comment:
No need for the `*` here. I know this is making me sound wishy-washy but I just can't imagine needing anything beyond a single arg here.

Reply:
reverted in 97698c7